skymap_scanner/config.py

```diff
@@ -17,6 +17,7 @@
 
 
 # physics strings
+INPUT_PULSES_NAME: Final = "SplitUncleanedInIcePulses"
 INPUT_TIME_NAME: Final = "HESE_VHESelfVetoVertexTime"
 INPUT_POS_NAME: Final = "HESE_VHESelfVetoVertexPos"
 OUTPUT_PARTICLE_NAME: Final = "MillipedeSeedParticle"
```

Comment:
At some point it might make sense to move these configurations into the reco-specific modules.

Reply:
This is a great idea

Reply:
Follow up in #173